packages/ui-map-components/src/components/Markers/disasterIcons/Cyclone.tsx

```diff
@@ -2,9 +2,9 @@
  * Tupaia
  *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
  */
-import React from 'react';
```

Comment:
We are not supporting the Disasters feature moving forward and that is the only use case for the disaster icons so let's remove all of them and any references to them in this package. thanks